src/app/docs/alert/alert-docs.component.html

```diff
@@ -4,10 +4,269 @@
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don’t use alerts to convey a normal lifecycle status of a record or page. Use <a href="https://developer.blackbaud.com/skyux/components/label">labels</a> or <a href="https://developer.blackbaud.com/skyux-theme/docs/status-indicator?docs-active-tab=design">status indicators</a> instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-3.png"
+          caption="Don't use alerts to highlight statuses on records or pages."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-4.png"
+          caption="Do use labels or status indicators to convey record status."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+      >
+        <p>
+            Don’t use alerts to show messages about the success of failure of user actions. Use <a href="https://developer.blackbaud.com/skyux/components/toast">toasts</a> instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-5.png"
+          caption="Don't use alerts to highlight the outcomes of user actions."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-6.png"
+          caption="Do use toasts to call out the outcomes of user actions."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/alert/alert-anatomy.png"
+      >
+        <sky-docs-anatomy-item>
+          Alert icon
+        </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>
+          Text
+        </sky-docs-anatomy-item>
+        <sky-docs-anatomy-item
+          isOptional="true"
+        >
+          Close button
+        </sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      
+        <sky-docs-design-guideline
+        heading="Alert type"
+      >
+        <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-1.png"
```

Comment:
In this image, I think the first sentence should be a complete sentence. And as mentioned above, I don't think we should use "click here" as sample link text. I'd change this to something more like: "The agreement expired. To continue, renew the agreement." where "renew the agreement" is the link text.

Also, is "agreement" the right word here? It feels slightly off to me, but I'm not sure. Should it maybe have a modifier like "license agreement"?